.github/workflows/main.yml

```diff
@@ -189,9 +207,21 @@ jobs:
           cd /tmp/xcb-errors
           sudo make install
 
+      # Only LGI git master supports Lua 5.4. We need to build it from source.
+      - name: Build and Install LGI
+        if: matrix.lua_version == '5.4'
+        run: |
+          wget -O /tmp/lgi.zip https://github.com/pavouk/lgi/archive/refs/heads/master.zip
```

Comment:
We need to build LGI from their master because there is currently no release with the latest patch for Lua 5.4. This will need to be changed when a new release is available.

Reply:
> when a new release is available.

I admire your optimism. Please do not click on https://github.com/pavouk?tab=overview&from=2019-12-01&to=2019-12-31  :-)

Reply:
Well, https://github.com/lunarmodules exists. Also, don't you have admin rights? If so, can you add @aire-one?